pkg/ccl/schemachangerccl/testdata/end_to_end/create_index/create_index.definition

```diff
@@ -1,43 +1,43 @@
 setup
-CREATE TABLE defaultdb.t1 (id INT PRIMARY KEY, name VARCHAR(256), money INT)
+CREATE TABLE defaultdb.t (id INT PRIMARY KEY, name VARCHAR(256), money INT)
 ----
 
 stage-exec phase=PostCommitPhase stage=:
-INSERT INTO t VALUES($stageKey, 'a');
-INSERT INTO t VALUES($stageKey + 1, 'b');
-INSERT INTO t VALUES($stageKey + 2, 'c');
-DELETE FROM t WHERE v = 'a' and k=$stageKey;
-INSERT INTO t VALUES($stageKey, 'a');
-UPDATE t SET v='a' WHERE k > 0;
+INSERT INTO t VALUES ($stageKey, 'a');
+INSERT INTO t VALUES ($stageKey + 1, 'b');
+INSERT INTO t VALUES ($stageKey + 2, 'c');
+DELETE FROM t WHERE name = 'a' and id = $stageKey;
+INSERT INTO t VALUES ($stageKey, 'a');
+UPDATE t SET name = 'a' WHERE id > 0;
```

Comment:
This whole test definition needed some love in order to pass the `ExecuteWithDMLInjection` test.